### PR TITLE
Swag shop

### LINKED ISSUE #104: Add the shop to the website
2 options 
- simply add the link: https://alephium.myspreadshop.ch/
We need to agree where to add it. 
- Embed the shop directly to the website image

@mvaivre @nop33 do you have a preference? Is the second option a lot of work ?

### PR DESCRIPTION
**Pull request by mvaivre**
Fixes #104 

Take a look on https://www2.alephium.org

### CODE REVIEW

File: src/content/homepage.md [hunk 1]
```diff
@@ -292,6 +292,15 @@ todoListSection:
           complete: false
         - text: Grant Program V2
           complete: false
+shopSection:
+  title: Get our swag
+  subtitle: Shard your life a little 🤘
```
Comment (nop33) [inline src/content/homepage.md]: That's the only thing I am not sure of 😆

**Review by nop33 — APPROVED** [review]
👏

File: src/content/homepage.md (hunk 1 above)
Comment (mvaivre) [reply src/content/homepage.md]: Mouhahha

Comment (mvaivre) [reply src/content/homepage.md]: @MaudSimon Can you take a look at the wording? Consider my text as a placeholder.

**Pull request merged**